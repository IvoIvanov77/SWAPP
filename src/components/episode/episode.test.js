import React from 'react';
import { shallow } from 'enzyme';
import Episode from './episode.component';

it('should render Episode component', () => {
  expect(shallow(<Episode />)).toMatchSnapshot();
});
