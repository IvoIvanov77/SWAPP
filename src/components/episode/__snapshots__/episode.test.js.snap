// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Episode component 1`] = `ShallowWrapper {}`;
